### ISSUE TITLE
Git fatal checkout on Windows

### ISSUE THREAD

**Issue by facelessuser** [squidfunk/mkdocs-material#316]
## Description

Was trying to checkout my fork for a pull request on my Windows system and got:

```
mkdocs-material.git
Cloning into 'mkdocs-material'...
remote: Counting objects: 8420, done.
remote: Compressing objects: 100% (6/6), done.
remote: Total 8420 (delta 3), reused 0 (delta 0), pack-reused 8414
Receiving objects: 100% (8420/8420), 6.88 MiB | 0 bytes/s, done.
Resolving deltas: 100% (4785/4785), done.
fatal: cannot create directory at 'tests/visual/baseline/ci/md-nav--primary/md-n
av__item--nested/md-nav__link/:focus': Invalid argument
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'
```

Windows does not allow `:` in folder names.  So this limits contributions from people on windows.  Luckily I usually work on Mac, but these folders should probably get renamed.

### Expected behavior

git clone to work on Windows.

### Actual behavior

Git clone fails (or better phrased, the checkout part fails) on windows due to invalid Windows folder names.

### Steps to reproduce the bug

1. Git clone on Windows.  Checkout will fail.

### Package versions

* Material: tip of master

### System information

* OS: Windows, and probably only windows.

**Comment by squidfunk** [comment]
Ah thanks, that's bad. I will remove the visual regression test baseline anyways as I want to switch to something more robust.

**Comment by squidfunk** [comment]
Fixed in #325.
